pkg/network/node/healthcheck_ovs.go

```diff
@@ -0,0 +1,40 @@
+package node
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/openshift/sdn/pkg/util/ovsclient"
+	utilwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+)
+
+const (
+	ovsDialDefaultNetwork = "unix"
+	ovsDialDefaultAddress = "/var/run/openvswitch/db.sock"
+)
+
+func healthCheckOVS() error {
+	klog.Infof("[OVS healthcheck] starting OVS health check")
+	c, err := ovsclient.DialTimeout(ovsDialDefaultNetwork, ovsDialDefaultAddress, time.Minute)
+	if err != nil {
+		return fmt.Errorf("[OVS healthcheck] error connecting to OVS: %v", err)
+	}
+	if err := c.Ping(); err != nil {
+		if cErr := c.Close(); cErr != nil {
+			return fmt.Errorf("[OVS healthcheck] error pinging OVS, err: %v, and closing the connection, err: %v", err, cErr)
+		}
+		return fmt.Errorf("[OVS healthcheck] error pinging OVS, err: %v", err)
+	}
+	go utilwait.Until(func() {
```

Comment:
the `utilwait.Until` isn't doing anything here, since the function never returns. Just `go func() { ...`
(or if you don't like "bare" `go func() { ...`, then split out a `waitForDisconnect(c ovsclient.Client)` and then do `go waitForDisconnect(c)`)